layouts/partials/header.html

```diff
@@ -44,15 +44,15 @@
         <div class="main-section" data-auto-burger="primary">
          {{ range site.Menus.main }}
            <div class="nav-box">
-            <h3><a href="{{ .URL }}">{{ .Title }}</a></h3>
+            <h3><a href="{{ .URL }}" data-proofer-ignore>{{ .Title }}</a></h3>
            {{ .Post }}
             </div>
          {{ end }}
         </div>
         <div class="main-section" data-auto-burger="primary">
             <div class="left">
                 <h5 class="github-invite">{{ T "main_github_invite" }}</h5>
-                <a href="https://github.com/kubernetes/kubernetes" class="button" data-auto-burger-exclude>{{ T "main_github_view_on" }}</a>
+                <a href="https://github.com/kubernetes/kubernetes" class="button" data-auto-burger-exclude data-proofer-ignore>{{ T "main_github_view_on" }}</a>
```

Comment:
What stops us leaving this link in for checking?

Reply:
Nothing, really, especially since I'm not currently scanning external links. 've added it back in – at this stage I was adding `data-proofer-ignore` all over the place to see what worked ;)